src/source/PeerConnection/Rtcp.c

```diff
@@ -73,6 +78,15 @@ STATUS onRtcpPacket(PKvsPeerConnection pKvsPeerConnection, PBYTE pBuff, UINT32 b
                         rttPropDelayMsec = KVS_CONVERT_TIMESCALE(rttPropDelay, DLSR_TIMESCALE, 1000);
                         DLOGD("RTCP_PACKET_TYPE_RECEIVER_REPORT rttPropDelay %u msec", rttPropDelayMsec);
                     }
+
+                    MUTEX_LOCK(pTransceiver->sender.statsLock);
+                    pTransceiver->sender.remoteInboundStats.reportsReceived++;
+                    pTransceiver->sender.remoteInboundStats.fractionLost = (DOUBLE) fractionLost / 255.0;
```

Comment:
Same here. How can an application access this? I am assuming the application will call the getMetrics call periodically?

Reply:
see comment above
re `Metrics.c`